objcgen/driver.cs

```diff
@@ -59,6 +59,7 @@ public static int Main2 (params string [] args)
 			var os = new OptionSet {
 				{ "c|compile", "Compiles the generated output", v => embedder.CompileCode = true },
 				{ "e|extension", "Compiles the generated output as extension safe api", v => embedder.Extension = true },
+				{ "ne|nativeexception", "Compiles the generated output to throw native exceptions (Apple only)", v => embedder.NativeException = true },
```

Comment:
I'm not sure `ne` works correctly (would it interfere with an -n and an -e option?) I'd just go for the full version only (`"nativeexception", "Compiles the ..."`)